lambda/index.js

```diff
@@ -1,21 +1,40 @@
 const exec = require('child_process').exec;
+const queue = require('queue');
+
+function processRecord(record) {
+  const child = exec('./ruby_wrapper ' + "'" +  JSON.stringify(record) + "' '" + JSON.stringify(context) + "'", (result) => {
+    context.done(result);
+  });
+
+  child.stdout.on('data', function (data) {
+    data.split("\n").forEach(function(x) {
+      console.log(x);
```

Comment:
Which, to give a bit more context, is how we get logs/stats into DataDog and SumoLogic.